concepts/goal.scroll

```diff
@@ -3,7 +3,7 @@ import ../code/conceptPage.scroll
 id goal
 name Goal
 appeared 2022
-creators Marina Fernández de Retana
+creators Yon Fernández de Retana
```

Comment:
Ok, I changed `frundis.scroll` to "Yon" as well. Let me know if that was a mistake. Thanks!

Reply:
Yep, it was brought to my attention by Yon (I didn't know either) on an array language discord.